doc/providers/form.rst

```diff
@@ -53,12 +53,12 @@ Registering
         composer require symfony/form
 
     If you are going to use the validation extension with forms, you must also
-    add a dependency to the ``symfony/config`` and ``symfony/translation``
```

Comment:
it transitive since [v2.2.0](https://packagist.org/packages/symfony/validator#v2.2.0) of the Validator
 (I found out why I didn't need the translation provider, I've a custom `trans`-filter I my project to bypass the need for the `symfony/translation` package)